### DISCUSSION TITLE
Redir-Host 模式下，关闭DNS劫持为什么所有流量还是都经过了openclash

### DISCUSSION THREAD

**Discussion by Uiisland** [vernesong/OpenClash#3709] [Q&A]
小白求教，真心不懂，或者有什么办法只让Openclash监听某个端口进来的流量吗

**Comment by ghost** [comment]
两个办法

1 改防火墙，但自称小白了，就不推荐了

2 用`SRC-PORT`规则，`SRC-PORT,80,policy` 将任何源端口为 80 的数据包路由到 policy.

---

`关闭DNS劫持为什么所有流量还是都经过了openclash` 这个是你没另外的劫持DNS的工具，所以还是走了“公共”区域
